db/migrate/20240430110330_add_group_to_devices.rb

```diff
@@ -0,0 +1,5 @@
+class AddGroupToDevices < ActiveRecord::Migration[7.1]
+  def change
+    add_column :devices, :group, :integer
```

Comment:
Yep, that was my first thought as well, and my second thought was that I didn't want to be moving too much around in the database given that we're aiming to wrap up work on this by friday. If we get more time in future to work on it this would be something I'd want to implement.